docs/reference/release-notes/8.0.0.asciidoc

```diff
@@ -0,0 +1,699 @@
+
+Features/ILM+SLM::
+* Ensuring that the `ShrinkAction` does not hang if total shards per node is too low {es-pull}76732[#76732] (issue: {es-issue}44070[#44070])
+* Less verbose serialization of snapshot failure in SLM metadata {es-pull}80942[#80942] (issue: {es-issue}77466[#77466])
+
+Features/Indices APIs::
+* Fix `ComposableIndexTemplate` equals when `composed_of` is null {es-pull}80864[#80864]
+
+Features/Java High Level REST Client::
+* Fix HLRC compatibility with Java 8 {es-pull}74290[#74290] (issues: {es-issue}73910[#73910], {es-issue}74272[#74272], {es-issue}74289[#74289])
+* Avoid `StackOverflowError` due to regex alternate paths {es-pull}61259[#61259] (issue: {es-issue}60889[#60889])
+
+Geo::
+* Preprocess polygon rings before processing it for decomposition {es-pull}59501[#59501] (issues: {es-issue}54441[#54441], {es-issue}59386[#59386])
+
+Infra/Core::
+* Add searchable snapshot cache folder to `NodeEnvironment` {es-pull}66297[#66297] (issue: {es-issue}65725[#65725])
+* CLI tools: Write errors to stderr instead of stdout {es-pull}45586[#45586] (issue: {es-issue}43260[#43260])
+* Precompute `ParsedMediaType` for XContentType {es-pull}67409[#67409]
+* Prevent stack overflow in rounding {es-pull}80450[#80450]
+
+Infra/Logging::
+* Fix NPE when logging null values in JSON {es-pull}53715[#53715] (issue: {es-issue}46702[#46702])
+* Fix stats in slow logs to be a escaped JSON {es-pull}44642[#44642]
+* Populate data stream fields when `xOpaqueId` not provided {es-pull}62156[#62156]
+
+Infra/REST API::
+* Do not allow spaces within `MediaType's` parameters {es-pull}64650[#64650] (issue: {es-issue}51816[#51816])
+* Handle incorrect header values {es-pull}64708[#64708] (issues: {es-issue}51816[#51816], {es-issue}64689[#64689])
+* Ignore media ranges when parsing {es-pull}64721[#64721] (issues: {es-issue}51816[#51816], {es-issue}64689[#64689])
+* `RestController` should not consume request content {es-pull}44902[#44902] (issue: {es-issue}37504[#37504])
+* Handle exceptions thrown from `RestCompatibleVersionHelper` {es-pull}80253[#80253] (issues: {es-issue}78214[#78214], {es-issue}79060[#79060])
+
+Infra/Scripting::
+* Change compound assignment structure to support string concatenation {es-pull}61825[#61825]
+* Fixes casting in constant folding {es-pull}61508[#61508]
+* Several minor Painless fixes {es-pull}61594[#61594]
+* Fix duplicated allow lists upon script engine creation {es-pull}82820[#82820] (issue: {es-issue}82778[#82778])
+
+Infra/Settings::
+* Stricter `UpdateSettingsRequest` parsing on the REST layer {es-pull}79227[#79227] (issue: {es-issue}29268[#29268])
+* Set Auto expand replica on deprecation log data stream {es-pull}79226[#79226] (issue: {es-issue}78991[#78991])
+
+Ingest::
+* Adjust default geoip logging to be less verbose {es-pull}81404[#81404] (issue: {es-issue}81356[#81356])
+
+Machine Learning::
+* Handle null value of `FieldCapabilitiesResponse` {es-pull}64327[#64327]
```

Comment:
```suggestion
* Handle null value of `FieldCapabilitiesResponse` {es-pull}64327[#64327]
```

This one should have been labelled `>non-issue` as it was fixing a mistake in a different PR in the same release.

Reply:
Removed in https://github.com/elastic/elasticsearch/pull/83689/commits/b4fbb7ace5e3ebf3fe1a3b9f1685d7d5815a160c